### DISCUSSION TITLE
Add an 'angle_difference' method for finding the shortest distance between two angles

### DISCUSSION THREAD

**Discussion by Miziziziz** [godotengine/godot-proposals#8108] [Engine Core]
Many times when I'm making games I need to find the shortest difference between 2 angles, right now the simplest way to do this is:
`var angle_diff = Vector2.RIGHT.rotated(angle_1).angle_to(Vector2.RIGHT.rotated(angle_2))`

It would be nice to simplify this into one method call like:
`var angle_diff = angle_difference(angle_1, angle_2)`

**Comment by theraot** [comment]
See https://github.com/godotengine/godot/pull/80225

**Comment by theraot** [comment]
I believe this is simpler right now:

```
var angle_diff = wrapf(angle_2 - angle_1, -PI, PI)
```

**Comment by AThousandShips** [comment]
Closing as implemented in godotengine/godot#80225
